### DISCUSSION TITLE
zig 无法正确找到 winsdk 的头文件

### DISCUSSION THREAD

**Discussion by Ariaszzzhc** [zigcc/forum#149] [新手提问] [answered]
当编译到 msvc abi 时，zig 找不到 winrt 目录下的头文件

![image](https://github.com/user-attachments/assets/eb8595a3-4578-4d60-8176-803218110f4f)

这是 zig 的 bug 吗？

**Comment by jiacai2050** [comment]
不用 win 环境...

但你可以用 verbose 选项看看它默认找的 include 路径。

**Reply by Ariaszzzhc** [reply, marked as answer]
我似乎找到了答案
https://github.com/ziglang/zig/pull/3343

zig msvc 下 libc 只会考虑 `um` 和 `shared` 两个目录。

zig 对与 windows 的支持真的不尽人意，感觉还有很多路要走。而且默认采用 mingw 而不是 msvc 感觉是错误。在 windows 上，mingw 很多东西都没有办法正确编译。

**Reply by jiacai2050** [reply]
win 的支持本来就少，所以出现这种情况可以理解。
